gen/calc.py

```diff
@@ -86,6 +86,22 @@ def calculate_mesos_slave_modules_json(mesos_slave_modules):
     return json_multiline.replace('\n', '\n' + injected_indent)
 
 
+def calculate_mesos_slave_modules(dcos_overlay_enable):
```

Comment:
We need this in 1.8. So if the infrastructure for adding modules is ready in that time frame, we can discard this PR and just use that infrastructure. Will check with @karya0 on this.